### PR TITLE
RMT sync and mirroring timers elaborated (bsc#1220076)

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Are there any relevant issues/feature requests?

* bsc#1220076

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP5/openSUSE Leap 15.5
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Review by felixsch — APPROVED** [review]
lgtm :+1: 

**How I reviewed this pull request:**

- Checked if the changes makes sense from a RMT user perspective.

**As always, if you think I missed something I should review/test, please let me know**:rocket:

File: xml/rmt_mirroring.xml [hunk 1]
```diff
@@ -123,18 +123,23 @@
...
    Loaded: loaded (/usr/lib/systemd/system/rmt-server-sync.timer; enabled; vendor preset: disabled)
    Active: active (waiting) since Fri 2018-06-22 04:22:34 EDT; 2h 34min ago
   Trigger: Sat 2018-06-23 03:53:00 EDT; 20h left
 
 Jun 22 04:22:34 d31 systemd[1]: Started RMT Sync timer.</screen>
+    <para>
+      If the timer is not enabled or started, start it manually.
+    </para>
+
+<screen>&prompt.sudo;systemctl enable --now rmt-server-sync.timer</screen>
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/rmt_mirroring.xml:142]: not necessarily but it is better with the command, you are right :-)

Comment (tbazant) [inline xml/rmt_mirroring.xml:142]:
```suggestion
<screen>&prompt.sudo;<command>systemctl enable --now rmt-server-sync.timer</command></screen>
```

File: xml/rmt_mirroring.xml [hunk 2]
```diff
@@ -165,6 +170,11 @@ Jun 22 04:22:34 d31 systemd[1]: Started RMT Sync timer.</screen>
 
 Jun 22 04:22:34 d31 systemd[1]: Started RMT Mirror timer.</screen>
 
+    <para>
+      If the timer is not enabled or started, start it manually.
+    </para>
+
+<screen>&prompt.sudo;systemctl enable --now rmt-server-mirror.timer</screen>
```
Comment (tbazant) [inline xml/rmt_mirroring.xml:177]:
```suggestion
<screen>&prompt.sudo;<command>systemctl enable --now rmt-server-mirror.timer</command></screen>
```

**Review by tbazant — COMMENTED** [review]
made 2 suggestions for improvement

**Review by dariavladykina — COMMENTED** [review]
LGTM. Thanks!

**Pull request merged**